docs/configuration.rst

```diff
@@ -241,14 +241,14 @@ This can be configured in one of two ways.
 **Separate coin sections**
 
 This is an alternative layout for the coin config mentioned above. It provides the ability to change the minloansize per coin, but is otherwise identical in functionality.
-To use this configuration, make sure to comment out the line where coincfg is defined, then add a secion for each coin you wish to configure. These sections should come 
-at the end of the file, after the other options for the bot.
+To use this configuration, make sure to comment out the line where coincfg is defined, then add a section for each coin you wish to configure.
+.. warning:: These sections should come at the end of the file, after the other options for the bot.
```

Comment:
OK. Is there a way you're testing that when writing? I was just using vim, bit I'd like to have actually rendered what it'll look like in readthedocs. 

I suspect I'll run into this type of issue again.